.gitignore

```diff
@@ -6,4 +6,5 @@
 venv/
 .pytest_cache/
 **__pycache__/
-db.sqlite3
+.sqlite3
```

Comment:
The file `.gitignore` should not include database files like `.sqlite3` as per the checklist requirement: 'Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension)'.